### PR TITLE
385 subscriptions tab

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.ts [hunk 1]
```diff
@@ -8,11 +8,11 @@ import { WorkflowTarget } from '../../../types';
 
 const MAX_LABEL_LENGTH = 45;
 
-export enum SubscriptionTabIds {
-    GENERAL_TAB = 'general-id',
-    SERVICE_CONFIGURATION_TAB = 'service-configuration-id',
-    PROCESSES_TAB = 'processes-id',
-    RELATED_SUBSCRIPTIONS_TAB = 'related-subscriptions-id',
+export enum WfoSubscriptionDetailTab {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.ts:11]: I would say to only name components Wfo..Name not the types and enums -- except props

File: packages/orchestrator-ui-components/src/pages/subscriptions/WfoSubscriptionsListPage.tsx [hunk 2]
```diff
@@ -55,28 +48,25 @@ export const WfoSubscriptionsListPage = () => {
...
-    const sortOrder = getSortDirectionFromString(
-        dataDisplayParams.sortBy?.order,
-    );
-    const selectedSubscriptionsTab =
-        getSubscriptionsTabTypeFromString(activeTab);
-    if (!sortOrder || !selectedSubscriptionsTab) {
-        router.replace('/subscriptions');
-        return null;
-    }
+    const selectedTab = (): WfoSubscriptionListTab => {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/pages/subscriptions/WfoSubscriptionsListPage.tsx:54]: Suggesting to change to `getSelectedTab`

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.ts (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.ts:11]: Renamed

File: packages/orchestrator-ui-components/src/pages/subscriptions/WfoSubscriptionsListPage.tsx (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/pages/subscriptions/WfoSubscriptionsListPage.tsx:54]: Made it an IIFE so selectTab get's set right away

**Review by Georgi2704 — APPROVED** [review]

**Pull request merged**